bd-logger/src/test/logger_integration.rs

```diff
@@ -926,11 +918,104 @@ fn workflow_flush_buffers_action_emits_synthetic_log_and_uploads_buffer_and_star
   });
 }
 
+#[test]
+fn workflow_generate_log_to_histogram() {
```

Comment:
for this or a different test it would be nice to verify that we are setting session ID, timestamp, etc as expected on the generated log